challengeFive.js

```diff
@@ -0,0 +1,46 @@
+// First attempt 
+
+let cardNumber = "";
+
+function hideCardNumber(number){
+    const cardNumberString = number.toString();
+    const integers = cardNumberString.split("");
+    
+    for(let i = 0; i < integers.length - 4; i++) {
+        integers[i] = "*";
+        cardNumber += integers[i]
+    }
+   
+    for(let i = integers.length - 4; i < integers.length; i++){
+        cardNumber += integers[i]
+    }
+    console.log(cardNumber)
```

Comment:
This should be returned and not logged to the console. 

The function should return the cardNumber
and not log it to the console. The function should be pure and not have any side effects.